data/json/items/melee/swords_and_blades.json

```diff
@@ -916,7 +916,7 @@
     "price_postapoc": 250,
     "bashing": 1,
     "cutting": 25,
-    "to_hit": { "grip": "solid", "length": "short", "surface": "point", "balance": "good" },
+    "to_hit": { "grip": "solid", "length": "hand", "surface": "line", "balance": "good" },
```

Comment:
If we are saying that ceremonial weapons have a solid grip at best I think that should apply to all replica weapons as well.

Reply:
Some of them are plain too small to actually be used at all, but the friend I talked( a Sikh to make sure I got this fairly okay) said that if it's big enough to grab without the cover on then it will probably have a good grip. Once it's that big the handle is a lot more visible and the maker's don't want them to be crap. So, I wouldn't say that this specific tool should affect replicas, since I think the handle would be the easiest part to make well.

Reply:
I can agree to that. Your argument makes sense.